scripts/generate_tilesets.py

```diff
@@ -132,6 +99,11 @@ def check_aoi_tiles(aoi_tiles_gdf):
     except:
         raise Exception("IDs do not seem to be well-formatted. Here's how they must look like: (<integer 1>, <integer 2>, <integer 3>), e.g. (<x>, <y>, <z>).")
     
+    if not aoi_tiles_gdf['id'].str.startswith('(').all():
+        aoi_tiles_gdf['id']='('+aoi_tiles_gdf['id']
+    if not aoi_tiles_gdf['id'].str.endswith(')').all():
+        aoi_tiles_gdf['id']=aoi_tiles_gdf['id']+')'
+    
     return
```

Comment:
Puisque j'ai modifié le code pour que l'id soit corrigé si des parenthèses manquent, faudrait-il mettre `return aoi_tiles_gdf`?  Est-ce que cela risquerait de déranger les `raise Exceptions`?

Reply:
Well... should the `id ` be badly formatted, an exception would be raised and your code wouldn't be executed.

* Do we want the code to support IDs formatted as either "(<x>, <y>, <z>)" or "<x>, <y>, <z>"?
* In which case the ID would be formatted as "<x>, <y>, <z>"?

Reply:
In the current configuration, the script does not raise an exception when the id is formatted without the parenthesis. That why we should change the function to raise an error or add the parenthesis